modules/ROOT/pages/file/file-read.adoc

```diff
@@ -5,36 +5,149 @@ endif::[]
 :keywords: file, connector, read
 
 
+The File connector can read a file at any point in the flow, unlike in Mule 3, where the
+transport can only read files as a result of inbound endpoint polling.
 
-The File connector can read a file at any point in the flow, unlike in Mule 3, where the transport can only read files as a result of inbound endpoint polling.
+The only required parameter for the `read` operation is `path`, which is the *relative path* of the file
+you wish to read. This path is relative to the `working directory` defined in the configuration
+referenced by the `config-ref` parameter. To see an example of creating a configuration check
+xref:file/file-connector.adoc#connection_settings[this page]. If no configuration is referenced,
+the working directory defaults to the value of the `user.home` system property. If the system
+property is not set, the connector fails to initialize.
 
-Here is the syntax for reading a file:
+Here is an example for reading a file:
 
 [source,xml,linenums]
 ----
-<file:read path="#[path]"
-  lock="true|false"
-  outputEncoding="UTF-8"
-  outputMimeType="application/csv; headers=false" />
+<file:read config-ref="File_Config" path="filePath"/>
 ----
 
-The operation above reads the file in the given path. It returns a `MuleMessage` with the following attributes:
+The `read` operation returns a `MuleMessage` with the following attributes:
```

Comment:
this part is wrong, the read operation returns the content of the file on the payload and the metada on attributes,don't metion mule message here